spec/filters/dissect_spec.rb

```diff
@@ -156,6 +156,38 @@
     end
   end
 
+  describe "Integer datatype conversion, handle large integers" do
+    let(:config) do <<-CONFIG
+      filter {
+        dissect {
+          convert_datatype => {
+            "big_number" => "int"
+          }
+        }
+      }
+    CONFIG
+    end
+    sample("big_number" => "4394740425750718628") do
```

Comment:
@guyboertje see my comments in production code, maybe we want to support actual `BigInteger` here as well?
If you also think we should, then we need a test case with an at least 19 digit number here :)